### DISCUSSION TITLE
Include File Search in systemmenu search

### DISCUSSION THREAD

**Discussion by tuxianerDE** [elementary/applications-menu#532] [Ideas]
Hey,

first and foremost I love elementaryOS the look and feel but also the ease of integration for its users, especialyl the ones in the family that are not that tech savvy.

One item I noticed, when I use the menu search to look for an app,I think in the previous version I also saw files appearing containing the name in the file name (or even content?). Currently I can only search for files in the file explorer not in the general overview. Did I accidently deactive this feature or was this a deliberate decision from you to disable this feature? 

At least me personally it supported my productivity.

Cheers & at this stage have a safe and healthy journey into 2022
Tuxianer

**Comment by cassidyjames** [comment]
elementary OS has never had file search in the Applications Menu, though there's an open issue with a lot of discussion and relevant information:

https://github.com/elementary/applications-menu/issues/54

You might be thinking of the file bookmarks search which was recently added? You can search for anything that has been bookmarked in the Files app, and it will show up in the Applications Menu.

**Comment by tuxianerDE** [comment]
Hey, thanks for the reply I must have suffered from delusionsm could have sworn I have seen it. 

Looking at the linked discussion thread you are right, this is what I was looking for/thinking off. Will close this one and engage in the other then :)
